A persona and pathway chapter for the turing way community handbook (#1254)
alan-turing-institute/the-turing-way: book/website/community-handbook/persona.md

(ch-persona)=
# Personas & Pathways in _The Turing Way_

As a community-driven and open source project, we are aware that each of our participants will have different backgrounds, different cultures, different time zones, different levels of skills and very different lived experiences.

<!---
| ![4 people with diverse appearances are talking to each other, sharing ideas directly with each other and also sharing on social media.](../figures/Personas.png)|
|--------------------------------|
| People sharing their ideas through direct conversations and on social media. _The Turing Way_ project illustration by Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3695300 |
--->

```{figure} ../figures/personas.png
---
height: 400px
name: personas
alt: 4 people with diverse appearances are talking to each other, sharing ideas directly with each other and also sharing on social media.
---
People sharing their ideas through direct conversations and on social media. _The Turing Way_ project illustration by Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3695300
```

> Persona is a detail of an imaginary user or member, based on real-world observations and understandings of existing members or potential future members.

Reviewer: ```suggestion
A persona is a detail of an imaginary user or member, based on real-world observations and understandings of existing members or potential future members.
```

It is meaningful when paired with pathways through which these personas engage with a project.

> To understand how our contributors or users might interact with _The Turing way_, we have developed some personas and Pathways that we are sharing in this chapter.

Reviewer: ```suggestion
To understand how our contributors or users might interact with _The Turing Way_, we have developed some personas and pathways that we are sharing in this chapter.
```

> They aims to capture different personas of the community members to explore how they come in touch with this project, how they continue to engage and how can we take their experiences and expectations into account when planning our community aspects.

Reviewer: ```suggestion
They aim to capture different personas of the community members to explore how they come into contact with this project, how they continue to engage and how we can take their experiences and expectations into account when planning our community aspects.
```
